package.json

```diff
@@ -11,10 +11,12 @@
     "test": "test"
   },
   "scripts": {
-    "lint": "aegir lint",
+    "lint": "tslint --fix src/*",
```

Comment:
--fix is not ok, because CI might fix something and pass but it will never get commited.

Any reason for switching from aegir?

Reply:
When I simply had aegir for linting, it didn't lint properly and it showed on github. So, I specified the linter to use in order to get around that.